code/office-director/office-director.js

```diff
@@ -59,7 +59,17 @@ function bigButton(id, view_id, url, fa_icon, button_label, target_blank = false
     $( "<a id='" + id + "' class='big-button-container" + disabledClass + " href='" + url + "'"
       + newTab + "'><span><i class='fa fa-" + fa_icon + "'></i></span><span> " + button_label + "</span></a>" ).appendTo("#" + view_id);
   if (callback) callback();
-}
+}	
+//>>>HOME TAB BUTTONS
+$(document).on('knack-view-render.view_50', function(event, page) {
+  // create large button on the home page
```

Comment:
Could remove these generic comment lines on line 65 and 70 or move them above the handler, `$(document)` line and make them more specific.